event-streams/src/event-streams-v1.0-wd01.md

```diff
@@ -0,0 +1,547 @@
+of the [OASIS IPR Policy](https://www.oasis-open.org/policies-guidelines/ipr),
+the mode chosen when the Technical Committee was established. For information on
+whether any patents have been disclosed that may be essential to implementing
+this specification, and any offers of patent licensing terms, please refer to
+the Intellectual Property Rights section of the TC's web page
+([https://www.oasis-open.org/committees/amqp/ipr.php](https://www.oasis-open.org/committees/amqp/ipr.php)).
+
+## 1.2 Terminology
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in BCP 14 [[RFC2119](#rfc2119)] and
+[[RFC8174](#rfc8174)] when, and only when, they appear in all capitals, as shown
+here.
+
+## 1.3 Normative References
+
+###### [AMQP-v1.0]
+_OASIS Advanced Message Queuing Protocol (AMQP) Version 1.0 Part 0: Overview_.
+Edited by Robert Godfrey, David Ingham, and Rafael Schloming. Latest version:
+http://docs.oasis-open.org/amqp/core/v1.0/amqp-core-overview-v1.0.html.
+
+###### [FILTEX-v1.0]
+_AMQP Filter Expressions Version 1.0_.
+Edited by Clemens Vasters. Latest version:
+http://docs.oasis-open.org/amqp/filtex/v1.0/filtex-v1.0.docx
+###### [RFC2119]
+Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14,
+RFC 2119, DOI 10.17487/RFC2119, March 1997,
+http://www.rfc-editor.org/info/rfc2119.
+###### [RFC8174]
+Leiba, B., "Ambiguity of Uppercase vs Lowercase in RFC 2119 Key Words", BCP 14,
+RFC 8174, DOI 10.17487/RFC8174, May 2017,
+http://www.rfc-editor.org/info/rfc8174.
+
+## 1.4 Non-Normative References
+
+###### [RFC3552]
+Rescorla, E. and B. Korver, "Guidelines for Writing RFC Text on Security
+Considerations", BCP 72, RFC 3552, DOI 10.17487/RFC3552, July 2003,
+https://www.rfc-editor.org/info/rfc3552.
+
+# 2 Concepts and default behaviors
+
+The following terminology is used in this specification to clarify roles. These
+definitions are narrower than their use in the core AMQP specification.
+
+- `producer` is the sender of events into the event log.
+- `consumer` is the receiver of events from the event log.
+- `event log node` is where AMQP links of senders and receivers attach.
+
+
+## 2.1 Partition support
+
+An event log managed by the event stream node MAY be split into multiple
+parallel partitions that are reachable under the same network address. A single
+event log might also be shared by multiple AMQP containers/nodes that each have
+a separate network address and are each responsible for one or more partitions.
+
+The association between a producer/consumer and a partition MAY be chosen using
+link properties when the link is being attached. The bi-directional negotiation
+capabilities of AMQP apply here: A consumer MAY request being associated with a
+particular partition, but the responding event log node MAY also assign a
+partition if none has been requested.
```

Comment:
just to be sure I got it correctly, are we saying that it's one partition per link? If more partitions are handled by the same event log node, does the receiver have to attach more links?